src/Robo/Commands/Acsf/AcsfCommand.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Acquia\Blt\Robo\Commands\Acsf;
+
+use Acquia\Blt\Robo\BltTasks;
+use Robo\Contract\VerbosityThresholdInterface;
+
+/**
+ * Defines commands in the "acsf" namespace.
+ */
+class AcsfCommand extends BltTasks {
+
+  /**
+   * Initializes ACSF support for project.
+   *
+   * @command acsf:init
+   *
+   * @options acsf-package
+   */
+  public function acsfInitialize($options = ['acsf-package' => 'drupal/acsf:^1.33.0']) {
```

Comment:
I'd just make the version constraint the option. The `drupal/acsf` part should never change.